src/Adapter/MSTestAdapter.PlatformServices/Services/TestSourceHost.cs

```diff
@@ -102,14 +102,8 @@ internal TestSourceHost(string sourceFileName, IRunSettings? runSettings, IFrame
     /// </summary>
     public void SetupHost()
     {
-#if NETFRAMEWORK || NET
-        List<string> resolutionPaths = GetResolutionPaths(
-            _sourceFileName,
-#if NETFRAMEWORK
-            VSInstallationUtilities.IsCurrentProcessRunningInPortableMode());
-#else
-            false);
-#endif
+#if NET
```

Comment:
It was getting complex to read so I have split .NET logic from .NET FX logic. We end up with a little bit of duplicated code but a way better readability.